src/cli/bin.ts

```diff
@@ -1,91 +1,26 @@
-      "keep-last-successful-version":
-        "could not be migrated because errors occurred during their migration. The version obtained after the last successful migration step was copied as is into the migrated folder.",
-      "keep-going":
-        "had errors occurring during their migration. They were tentatively migrated to the desired version, but very likely have issues and won't work well in the UI.",
-    },
-    removed:
-      "were cleaned up because they could not be found in the ui/dashboards/structure folder. They were already not visible in your version of ActiveUI.",
-  },
-  filters: {
-    success: "were successfully migrated.",
-    failed: {
-      "keep-original":
-        "could not be migrated because errors occurred during their migration. Their original versions were copied as is into the migrated folder.",
-      "keep-last-successful-version":
-        "could not be migrated because errors occurred during their migration. The version obtained after the last successful migration step was copied as is into the migrated folder.",
-      "keep-going":
-        "had errors occurring during their migration. They were tentatively migrated to the desired version, but very likely have issues and won't work well in the UI.",
-    },
-    removed:
-      "were cleaned up because they could not be found in the ui/filters/structure folder. They were already not visible in your version of ActiveUI.",
-  },
-  widgets: {
-    success: "were successfully migrated.",
-    partial: "were migrated with warnings.",
-    removed:
-      "were cleaned up because they could not be found in the ui/widgets/structure folder or because their keys were passed in the --remove-widgets option.",
-    failed: {
-      "keep-original":
-        "could not be migrated because errors occurred during their migration. Their original versions were copied as is into the migrated folder.",
-      "keep-last-successful-version":
-        "could not be migrated because errors occurred during their migration. The version obtained after the last successful migration step was copied as is into the migrated folder.",
-      "keep-going":
-        "had errors occurring during their migration. They were tentatively migrated to the desired version, but very likely have issues and won't work well in the UI.",
-    },
-  },
-  folders: {
-    removed:
-      "were cleaned up because they could not be found in their structure folder. They were already not visible in your version of ActiveUI.",
-  },
-  calculated_measures: {
-    success: "were successfully migrated.",
-    failed:
-      "could not be migrated because errors occurred during their migration.",
-  },
-};
+export function getFileExtension(path: string): string {
+  const fileExtension = path.split(".").pop()?.toUpperCase();
+
+  if (!fileExtension || !supportedFileExtension.includes(fileExtension)) {
+    throw new Error(`File of type ${fileExtension} are not supported.`);
```

Comment:
Let's make the syntax more human and the message more actionable:

```suggestion
    throw new Error(`The extension "${fileExtension}" is not supported. Supported extensions for the input file are "JSON" and "IPYNB"`);
```